analyzer/src/main/resources/init.gradle

```diff
@@ -243,15 +252,15 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
                             dependencyProject.version ?: '', '', '', dependencies, null, '',
                             dependencyProject.projectDir.absolutePath)
                 } else {
-                    return new DependencyImpl('', normalizeName(id.displayName), '', '', '', dependencies,
-                            "Unknown id type: ${id.getClass().simpleName}".toString(), '', null)
+                    return dependencyFromDisplayName(id.displayName, dependencies,
```

Comment:
Are the linke breaks here and below still required?

Reply:
Yes, it would otherwise still exceed 120 columns.